platformio.ini

```diff
@@ -15,11 +15,11 @@ board_f_cpu = 160000000L
 lib_deps_builtin =
   SPI
 lib_deps_external =
-  sidoh/WiFiManager#cmidgley
+  WiFiManager=https://github.com/sidoh/WiFiManager.git#cmidgley
   RF24@~1.3.2
   ArduinoJson@~6.10.1
   PubSubClient@~2.7
-  ratkins/RGBConverter@07010f2
+  https://github.com/ratkins/RGBConverter.git
```

Comment:
Can we fix this to the same commit? Hoping the same syntax works `@07010f2`. I prefer to have all dependencies locked to a particular version.